linux/net/rina/ipcps/shim-eth-vlan.c

```diff
@@ -51,6 +51,8 @@
 #define DEFAULT_QDISC_MAX_SIZE 50
 #define DEFAULT_QDISC_ENABLE_SIZE 10
 
+#define BLOCKING_FLOW 1
```

Comment:
Blocking or non blocking is a property of flows consumed by apps (via the
read/write syscalls implented in the KFA), not of flows used by IPCPs.
El 17/08/2015 10:38, "Sander Vrijders" <notifications@github.com> escribió:

> In linux/net/rina/ipcps/shim-eth-vlan.c
> <https://github.com/IRATI/stack/pull/713#discussion_r37167575>:
>
> > @@ -51,6 +51,8 @@
> >  #define DEFAULT_QDISC_MAX_SIZE 50
> >  #define DEFAULT_QDISC_ENABLE_SIZE 10
> >
> > +#define BLOCKING_FLOW 1
>
> Can you elaborate why this is needed? Is this needed in every shim? Then
> probably it should be defined in a common place (common.h)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/IRATI/stack/pull/713/files#r37167575>.
>